my_calculator.js

```diff
@@ -0,0 +1,44 @@
+// This is my "starter code". I generated it from scatch.
+
+//Function is math. When his function is called, a function named 'addition' is run with two arguments being passed through it. When num1 and num2 are passed through, it returns and output of that function for a sum of num1 and num2.
+
+// use method math.add(1,3).
+// Syntax error on line 8...
+
+var calc = {
+  value: 0,
+  add: function(addnum){
+    this.value += addnum;
+    this.showValue();
+  },
+  sub: function(subnum){
+    this.value -= subnum;
+    this.showValue();
+  },
```

Comment:
The way each of these methods work is by updating `calc.value` each of these methods are properties of the object `calc` and can be referenced by saying `calc.someMethod` - e.g. `calc.sub(arugment)`. The `this` can mean a lot of different things depending on where it is used, here `this` refers to the object they are inside of here (`calc`)